src/platform.cpp

```diff
@@ -3,6 +3,7 @@
 #include <filesystem>
 #include <fstream>
 #include <string>
+#include <unistd.h>
```

Comment:
could you put some #ifs around this so its only included by what needs it? win32 doesnt have this header for example

Reply:
Yes, sure. Will do.

Reply:
@ouwou Should be good now, I guess.